dashboard/test/ui/browsers.json

```diff
@@ -51,20 +51,22 @@
   },
   {
     "name": "iPhone",
-    "browserName": "iphone",
-    "version": "8.2",
+    "platformName": "iOS",
+    "platformVersion": "9.2",
+    "browserName": "safari",
     "deviceName": "iPhone Simulator",
-    "platform": "OS X 10.10",
+    "appiumVersion": "1.4.16",
     "mobile": true,
     "deviceOrientation": "landscape",
     "rotatable": true
```

Comment:
I think in general they're in a loose logical order: name first, then platform, etc. Beyond that maybe it's arbitrary and could use a reorder. At the very least the name first makes sense. 

Also I kept the order from SauceLabs' [example](https://wiki.saucelabs.com/display/DOCS/Examples+of+Desired+Capabilities+for+iWebDriver+and+Appium+iOS+Tests)